domain/repository/repository_service.go

```diff
@@ -56,5 +65,26 @@ func CreateService(urlToParse string, branch *string, configuration config.Confi
 		return nil, github.ErrUnsupportedGithubURL
 	}
 
-	return github.Create(urlSegments, branch, configuration)
+	url, err := url.Parse(configuration.GetAuthServiceURL())
+	if err != nil {
+		return nil, errors.Wrap(err, "auth service url not found")
+	}
+
+	authClient := client.New(goaclient.HTTPClientDoer(http.DefaultClient))
+	authClient.Host = url.Host
+	authClient.Scheme = url.Scheme
+	if goajwt.ContextJWT(*ctx) != nil {
+		authClient.SetJWTSigner(goasupport.NewForwardSigner(*ctx))
+	} else {
+		log.Logger().Info(ctx, nil, "no token in context")
+	}
+	tokenRetriever := tr.TokenRetriever{AuthClient: authClient, Context: ctx}
+	token, err := tokenRetriever.TokenForService(fmt.Sprintf("https://%s", githubHost))
+	if err != nil {
+		return nil, errors.Wrap(err, "auth service url not found")
+	}
+	if token == nil {
+		return nil, errors.Wrap(err, "token not found for GitHub")
```

Comment:
Since this may not be github in the future, i think it's better to use "u.Host" instead

Reply:
@corinnekrych You missed this one :)